### ISSUE TITLE
is it relevant to translate the ofSite to Portuguese? [i can do it]

### ISSUE THREAD

**Issue by tiagomoraismorgado88** [openframeworks/ofSite#630]
is it relevant to translate the ofSite to Portuguese? [i can do it]

**Comment by tmm2018** [comment]
again, i can do it

**Comment by arturoc** [comment]
yes a new translation is always welcome, let me know if you have any doubt on how translating works. what we don't have in place is a way to translate the documentation yet.
